test/evergreen.yml

```diff
@@ -413,14 +421,6 @@ functions:
         set -o errexit
         set -o verbose
         ${test_env_vars|} ./t ${thread_test_args|} 2>&1
-  "random abort test":
-    command: shell.exec
-    params:
-      working_dir: "wiredtiger/build_posix/test/csuite"
-      script: |
-        set -o errexit
-        set -o verbose
-        ${test_env_vars|} ./test_random_abort ${random_abort_args|} 2>&1
   "tiered abort test":
```

Comment:
Can the following abort test functions (tiered, timstamp, etc) be deprecated in favor of the new `csuite test` function?

Reply:
Some of them are actually unused and we are deciding what to do with them as part of WT-8504.